lib/code_corps/stripe_service/validators/project_can_enable_donations.ex

```diff
@@ -17,8 +17,10 @@ defmodule CodeCorps.StripeService.Validators.ProjectCanEnableDonations do
 
   and only one of:
 
-  - `Organization` with a `StripeConnectAccount`, in production env, which has `charges_enabled: true`
-  - `Organization` with a `StripeConnectAccount`, not in production env
+  - `Organization` with a `StripeConnectAccount`, in production env, which
+    has `charges_enabled: true` and `transfers_enabled: true`
+  - `Organization` with a `StripeConnectAccount` and
+    `transfers_enabled: true`, not in production env
```

Comment:
@begedin I think it's because `charges_enabled` is always `true` in some environments? Not sure.